lib/grant.js

```diff
@@ -140,6 +141,9 @@ function checkClient (done) {
       return done(error('invalid_client', 'Client credentials are invalid'));
     }
 
+    self.client.accessTokenLifetime = client.accessTokenLifetime;
+    self.client.refreshTokenLifetime = client.refreshTokenLifetime;
```

Comment:
Ah yes, that makes sense!

OK, let's keep it as is at the moment, I want to have a play with how client/creds are stored for 3.0.0 anyway (see #144)